Foundation/src/DateTime.cpp

```diff
@@ -154,6 +136,7 @@ DateTime& DateTime::operator = (const Timestamp& timestamp)
 	_utcTime = timestamp.utcTime();
 	computeGregorian(julianDay());
 	computeDaytime();
+	checkValid();
```

Comment:
Is this check intended to verify the conversion of `Timestamp` to `DateTime` because this is the only way that `DateTime` can be invalid.

Reply:
yes